documentation/protocol.md

```
@@ -0,0 +1,153 @@
+## 🛠️ Fonctions Disponibles
+
+| Fonction | Description | Paramètre d'entrée | Valeur de retour |
+|----------|-------------|-------------------|------------------|
+| `encrypt` | Chiffre un message en texte clair | message (str) - Texte à chiffrer | str - Message chiffré |
+| `decrypt` | Déchiffre un message chiffré | message (str) - Texte chiffré | str - Message en texte clair |
+
+---
+
+## 🔄 Processus de Communication
+
+### Étapes du processus :
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ' :'

(MD026, no-trailing-punctuation)